tutorials/introductory/quilt_layout.py

```diff
@@ -0,0 +1,252 @@
+"""
+==============
+Patchwork Axes
+==============
+
+This is inspired by the `patchwork
```

Comment:
Please re-itterate that this is provisional API like is done in the docstring.

Reply:
Same as above, `named` or `semantic` axes, not patchwork, and the tutorial should be named something like `semantic_layout.py`